tests/test_transport_network.py

```diff
@@ -76,3 +79,27 @@ def test_street_layer_java_object(self, transport_network):
     def test_timezone(self, transport_network, gtfs_timezone_helsinki):
         assert isinstance(transport_network.timezone, java.time.ZoneId)
         assert transport_network.timezone.toString() == gtfs_timezone_helsinki
+
+    def test_cache_directory(self, transport_network_files_tuple):
```

Comment:
Cannot use the already generated `transport_network_from_files` fixture because that seems to have some persistent links: garbage collection does not kick in, `__del__()` is not called within the test scope.

Reply:
👍🏻